### DISCUSSION TITLE
Is there a way to install multiple versions of Haxe?

### DISCUSSION THREAD

**Discussion by Coquette-Shadow** [YoshiCrafter29/YoshiCrafterEngine#145] [Q&A] [answered]
Is there a way to install and switch between multiple versions of haxe. I need 4.2.4 for Psych Engine but 4.1.5 for all others!

**Comment by YoshiCrafter29** [comment, marked as answer]
It works too with latest version, so no need

ninjamuffin said 4.1.5 cause the latest version at the time he said it had problems with git

since then the problems were fixed

**Reply by Coquette-Shadow** [reply]
Yoooo thanks for the answer and for the amazing engine!
